hack/make/.go-autogen

```diff
@@ -2,20 +2,23 @@
 
 rm -rf autogen
 
-mkdir -p autogen/dockerversion
-cat > autogen/dockerversion/dockerversion.go <<DVEOF
-// AUTOGENERATED FILE; see $BASH_SOURCE
+cat > dockerversion/version_autogen.go <<DVEOF
+// +build autogen
+// Package version is auto-generated at build-time
 package dockerversion
 
-var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
-	BUILDTIME string = "$BUILDTIME"
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GitCommit string = "$GITCOMMIT"
+	Version   string = "$VERSION"
+	BuildTime string = "$BUILDTIME"
 
-	IAMSTATIC string = "${IAMSTATIC:-true}"
-	INITSHA1  string = "$DOCKER_INITSHA1"
-	INITPATH  string = "$DOCKER_INITPATH"
+	IAmStatic string = "${IAMSTATIC:-true}"
```

Comment:
should this check the env variable in camel case beside in all capitals? I guess we left it like that to be backwards compatible, but should we also be consistent?